Bottom: be31f984b5ec66e45ccfc5086e4c1901ede2f3ef
Top:    44d098d5da6c639e790e5ad559b225305cf33349
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2014-02-25 19:43:37 -0500

find.c: The User varaible needs to die...

Reduce its usage to where absolutely necessary... oh, what a surprise,
another variable passing state down into add_object_to_list ()


---

diff --git a/src/find.c b/src/find.c
index 3747233..1c755f7 100644
--- a/src/find.c
+++ b/src/find.c
@@ -3090,7 +3090,7 @@ LookupConnection (Coord X, Coord Y, bool AndDraw, Coord Range, int flag,
    */
   ListStart (type, ptr1, ptr2, ptr3, flag);
   DoIt (flag, AndRats, AndDraw);
-  if (User)
+  if (AndDraw)
     IncrementUndoSerialNumber ();
   User = false;
